addons/first_heroes_win.wad/init.lua

```diff
@@ -0,0 +1,188 @@
+include "scripting/coroutine.lua"
+include "scripting/messages.lua"
+include "scripting/richtext.lua"
+include "scripting/win_conditions/win_condition_functions.lua"
+include "scripting/win_conditions/win_condition_texts.lua"
+
+local debug = false
+local function d_print(text)
+   if debug then print(text) end
+end
+local function t_print(tbl)
+   if debug then
+      local t = ""
+      for i,v in ipairs(tbl) do
+         t = t .. v .. ", "
+      end
+      return "{"..t.."}"
+   end
+   return ""
+end
+
+-- If changing this value remember to change it also in the addons-file
+local MAXHEROES = 4
+
+local textdomain = "first_heroes_win.wad"
+
+push_textdomain(textdomain, true)
+
+local wc_name = "First Heroes Win"
+-- This needs to be exactly like wc_name, but localized, because wc_name
+-- will be used as the key to fetch the translation in C++
+local wc_descname = _("First Heroes Win")
+local wc_version = 1
+local wc_desc = _("The player who has at least %1% fully trained soldiers in his warehouses wins the game."):bformat(ngettext("%1%", "%1%", MAXHEROES))
+local wc_first_hero = "Number of heroes"
+-- This needs to be exactly like wc_first_hero, but localized, because wc_first_hero
+-- will be used as the key to fetch the translation in C++
+local wc_first_hero_i18n = _("Number of heroes")
+
+local r = {
+   name = wc_name,
+   description = wc_desc,
+   peaceful_mode_allowed = true,
```

Comment:
```suggestion
   peaceful_mode_allowed = true,
   textdomain = textdomain,
```

Reply:
Can you explain please?

Reply:
When you load a savegame, the localized win condition name to be shown in the UI is fetched from the textdomain which is specified by this table. If this key is absent, the official `win_conditions` textdomain is used; add-ons need to specify their own textdomain here so the strings are still localized in the Load Game screen.

Reply:
https://www.widelands.org/documentation/add-ons/#win-condition